src/config.ts

```diff
@@ -0,0 +1,14 @@
+import { registerAs } from '@nestjs/config';
+
+export default registerAs('config', () => ({
+    database: {
+        dbName: process.env.MYSQL_DB,
+        port: parseInt(process.env.DB_PORT),
+        password: process.env.DB_PASSWORD,
+        user: process.env.DB_USER,
+        host: process.env.DB_HOST,
+    },
+    MYSQL_URL: process.env.DB_URL,
+    // jwtSecret: process.env.JWT_KEY,
```

Comment:
Esta y la de abajo se usarán?

Reply:
sii pero mas adelante, no las borre por que no queria olvidar en que parte del code irian pero debi borrarlas

Reply:
Va, las mejores prácticas indican agregar un comentario así:
// TO-DO: Implementar cuando se trabaje en BR-X (ID del ticket)

Reply:
okey! gracias